homeassistant/components/obihai/obihai_api.py

```diff
@@ -0,0 +1,31 @@
+"""Support for Obihai Sensors."""
```

Comment:
I think you should rename this file - we know it's `obihai` already from the domain.
```suggestion
"""Support for Obihai connectivity."""
```